faq.html

```diff
@@ -1,463 +1,339 @@
-                                                kontakta eduID-supporten via e-post till <a href="mailto:support@eduid.se">support@eduid.se</a>.
-                                            </p>
-                                            <p>
-                                                Ange alltid den e-postadress som du använde när du loggade in på eduID. Om något
-                                                har blivit fel är det alltid bra att skicka med skärmdumpar med felmeddelanden
-                                                för att underlätta felsökning.
-                                            </p>
-                                            <p>
-                                                För bästa möjliga support rekommenderar vi dig alltid att skicka e-post men
-                                                för enklare ärenden kan du också nå oss per telefon 0455-385200.<br>
-                                                (Öppet måndag till torsdag 09:00-12:00, 13:00-16:00, fredag 9:00-12:00, 13:00-14:30)<br>
-                                            </p>
-                                            <p>
-                                                <strong>
-                                                    Telefonsupporten är stängd mellan den 8:e juli och den 4:e augusti 2019.<br>
-                                                    Support via e-post fungerar som vanligt under ovanstående period.
-                                                </strong>
-                                            </p>
-                                        </li>
-                                    </ul>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
+          <li>
+            <h2>
+              Hur hittar jag ett ATG ombud som kan göra en legitimationskontroll
+              för Freja eID?
+            </h2>
+            <p>
+              Ditt närmsta ATG ombud kan hittas i appen.
+            </p>
+          </li>
 
-            <div class="button-holder"><button class="button scroll-down"></button></div>
-        </div>
-    </section>
+          <li>
+            <h2>
+              Vad ska jag göra om legitimationskontrollen för Freja eID
+              misslyckades?
+            </h2>
+            <p>
+              Avinstallera appen, gör om registreringen, samt kontrollera noga
+              att du angivit rätt datum då ID-handlingen upphör att gälla samt
+              att du fyllt i rätt referensnummer och personnummer.
+            </p>
+          </li>
 
+          <li>
+            <h2>
+              Hur använder jag Freja eID med eduID?
+            </h2>
+            <p>
+              Logga in på eduID och klicka på 'Använd mitt Freja eID'. Du kan
```

Comment:
The second sentence here is no longer true. To verify you account with the user needs to have Freja eID+ before starting the verification.